examples/demo/gloo/terrain.py

```diff
@@ -25,22 +25,22 @@ def generate_terrain(r_min, r_max, c_min, c_max, disp):
     """Recursively generates terrain using diamond-square algorithm
     and stores the vertices in points
     """
-    a = points[r_min][c_min][2]
-    b = points[r_min][c_max][2]
-    c = points[r_max][c_min][2]
-    d = points[r_max][c_max][2]
+    a = points[r_min, c_min, 2]
+    b = points[r_min, c_max, 2]
+    c = points[r_max, c_min, 2]
+    d = points[r_max, c_max, 2]
 
-    r_mid = (r_min + r_max)/2
-    c_mid = (c_min + c_max)/2
+    r_mid = int((r_min + r_max)/2)
```

Comment:
can just be `(r_min + r_max) // 2` for integer division, assuming `r_min` and `r_max` are integers (if they aren't then what you have is fine)

Reply:
Ok. It's definitely faster.

```python
In [1]: x= 3.9

In [2]: %timeit x//2
233 ns ± 1.15 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [3]: %timeit int(x/2)
293 ns ± 2.95 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```